src/components/collapsible-motion/collapsible-motion.js

```diff
@@ -118,17 +117,37 @@ class CollapsibleMotion extends React.PureComponent {
         {({ isOpen, toggle }) => (
           <ToggleAnimation isOpen={isOpen} toggle={toggle}>
             {({
+              animation,
               containerStyles,
               toggle: animationToggle,
               registerContentNode,
-            }) =>
-              this.props.children({
-                isOpen,
-                toggle: animationToggle,
-                containerStyles,
-                registerContentNode,
-              })
-            }
+            }) => (
+              <ClassNames>
+                {({ css }) => {
+                  let animationStyle = {};
+
+                  if (animation) {
+                    // this is required so that the keyframes animation is actually injected into the DOM
+                    css`
```

Comment:
as comment says, without this we don't inject anything into the dom re: keyframes

Reply:
Could you explain what "this" refers to in the comment? The `css` tagged template literal? Maybe you could clarify the comment.

Reply:
Sure, calling css here returns a className, and injects the styles into the head. Without this, we don't end up with the keyframes in the document and the animation doesn't work. I think we have to do this weird workaround because generally you don't write components using emotion like this.